README.md

```diff
@@ -16,29 +16,46 @@ OpenNeuro is a free and open platform for analyzing and sharing neuroimaging dat
 This project is managed with [Lerna](https://lernajs.io/) and [Yarn](https://yarnpkg.com/). To get started, install Yarn and bootstrap the repo.
 
 ```shell
-yarn install && yarn bootstrap
+yarn install
 ```
 
 You can run tests with `yarn test` at the top level of the project. For each package, `yarn test --watch` will interactively run the tests for changes since the last commit.
 
-Before starting up the services, you will need to copy the example `.env.example` file to `.env` and `config.env.example` to `config.env`. Many of the values are optional, and most that aren't have default values included in their `.example` file. The exception is `JWT_SECRET` in `config.env`, which you will need to set to a large random string.
+Before starting up the services, you will need to copy the example `.env.example` file to `.env` and `config.env.example` to `config.env`. Many of the values are optional, and most that aren't have default values included in their `.example` file. Required values below:
+
```

Comment:
I had to set `CRN_SERVER_URL` in `config.env` to `http://localhost:9876` in order to get front end to fully load, may be worth mentioning here, or setting as default in example.

Reply:
Went ahead and added it as the default value.